hyperjumptech/monika
# Test: 526 add bats cli test

# Monika Pull Request (PR)  

## What feature/issue does this PR add  
1. Need to add automation test engine for monika

## How did you implement / how did you fix it  
1.  By adding bats test engine
2. Add initial checking to run the test

## How to test  
1. install dependencies: `npm ci`
2. run `npm run cli-test`
3. make sure the tests is passed
